uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/components/staff-leave-report.js

```diff
@@ -203,38 +165,10 @@ define([
     }
 
     /**
-     * Checks if the given leave request can be cancelled
-     *
-     * Based on following constants
-     * REQUEST_CANCELATION_NO = 1;
-     * REQUEST_CANCELATION_ALWAYS = 2;
-     * REQUEST_CANCELATION_IN_ADVANCE_OF_START_DATE = 3;
-     *
-     * @param  {LeaveRequestInstance} leaveRequest
-     * @return {Boolean}
+     * Clears the cached data of all sections
      */
-    function canLeaveRequestBeCancelled (leaveRequest) {
-      var allowCancellationValue = vm.absenceTypes[leaveRequest.type_id].allow_request_cancelation;
-
-      if (role === 'admin') {
-        return true;
-      }
-
-      if (allowCancellationValue === '3') {
-        return moment().isBefore(leaveRequest.from_date);
-      }
-
-      return allowCancellationValue === '2';
-    }
-
-    /**
-     * Clears the cached data of all the closed sections
-     */
-    function clearClosedSectionsData () {
+    function clearSectionsData () {
```

Comment:
Because it now flushes all sections, instead of open sections only. This is needed to ensure Angular to refresh actions dropdown. It is used only in vm.refresh function and since data is refreshed in all sections anyway, this is safe and makes sense.